api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java

```diff
@@ -0,0 +1,29 @@
+package org.zalando.nakadi.model;
+
+import org.apache.commons.lang.StringUtils;
+import org.zalando.nakadi.domain.ResourceAuthorizationAttribute;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationAttribute;
+
+import java.beans.PropertyEditorSupport;
+
+public class AuthorizationAttributeQueryParser extends PropertyEditorSupport {
+
+    @Override
+    public void setAsText(final String text) throws IllegalArgumentException {
+        if (StringUtils.isEmpty(text)) {
+            throw new IllegalArgumentException("Authorization query is empty");
+        } else {
+            final String[] authorizationQuery = text.split(":");
+            if (authorizationQuery.length != 2) {
+                throw new IllegalArgumentException("Authorization format is incorrect. Should be data_type:data_value");
```

Comment:
```suggestion
                throw new IllegalArgumentException("Authorization format is incorrect. Should be data_type:value");
```
Matches the API definition better.